jenkins/ompi/ompi_test.sh

```diff
@@ -110,265 +103,226 @@ mlnx_cov="SKIP"
 cov_exclude_file_list="oshmem/mca/memheap/ptmalloc/malloc.c"
 
 # global mpirun options
-export OMPI_MCA_mpi_warn_on_fork=0
-export OMPI_MCA_btl_openib_warn_default_gid_prefix=0
+export OMPI_MCA_mpi_warn_on_fork="0"
+export OMPI_MCA_btl_openib_warn_default_gid_prefix="0"
 
 # exclude hwloc external package
-if [ -d "opal/mca/hwloc/hwloc" ]; then
-    for excl in $(find opal/mca/hwloc/hwloc* -type f -name "*.[ch]"); do
-        cov_exclude_file_list="$cov_exclude_file_list $excl"
+if [ -d "opal/mca/hwloc/hwloc" ]
+then
+    # shellcheck disable=SC2044
+    for excl in $(find opal/mca/hwloc/hwloc* -type f -name "*.[ch]")
+    do
+        cov_exclude_file_list="${cov_exclude_file_list} $excl"
     done
 fi
 
-gh_cov_msg="$WORKSPACE/cov_file_${BUILD_NUMBER}.txt"
-OMPI_HOME1=$WORKSPACE/ompi_install1
-ompi_home_list="$OMPI_HOME1"
-topdir=$WORKSPACE/rpms
-tarball_dir=${WORKSPACE}/tarball
+gh_cov_msg="$WORKSPACE/cov_file.txt"
+OMPI_HOME1="$WORKSPACE/ompi_install1"
+ompi_home_list="${OMPI_HOME1}"
```

Comment:
ompi_home_list variable can be remove, no need OMPI_HOME1
we have only ONE OMPI_HOME

Reply:
Fixed.